backend/associations/serializers/election.py

```diff
@@ -55,16 +55,13 @@ class Meta:
 class ElectionChoiceSerializer(serializers.ModelSerializer):
     class Meta:
         model = Choice
-        fields = ("id", "name", "number_of_offline_votes")
-
-    def to_representation(self, instance: Choice):
-        response = super(ElectionChoiceSerializer, self).to_representation(instance)
-
-        if instance.election.show_results:
-            response["number_of_online_votes"] = instance.number_of_online_votes
-            response["number_of_votes"] = instance.number_of_votes
-
-        return response
+        fields = (
+            "id",
+            "name",
+            "number_of_offline_votes",
+            "number_of_online_votes",
+            "number_of_votes",
+        )
```

Comment:
Before you ask, this is necessary to allow the organiser see the results before they publish them